test/WebSites/FormatterWebSite/Controllers/JsonFormatterController.cs

```diff
@@ -21,8 +22,11 @@ public IActionResult ReturnsIndentedJson()
                 Name = "John Williams"
             };
 
-            var jsonFormatter = new JsonOutputFormatter();
-            jsonFormatter.SerializerSettings.Formatting = Formatting.Indented;
+            var settings = new JsonSerializerSettings
+            {
+                Formatting = Formatting.Indented,
+            };
+            var jsonFormatter = new JsonOutputFormatter(settings, ArrayPool<char>.Shared);
```

Comment:
This is a fairly common scenario. Users needing to know about `ArrayPool` is probably not a good experience.